views/add_event.hbs

```diff
@@ -0,0 +1,113 @@
+<div class="add-eventContainer">
+  <h3 class="userAddEvent">add event</h3>
+
+  <form class="add-event-form" action="/add-event" method="post">
+    <label>Title: </label>
+    <input class="form-control" type="text" name="name" placeholder="title" required>
+
+    <label> Image URL: </label>
+    <input class="form-control" type="text" name="imageUrl" placeholder="image URL link">
+
+    <label>Start Date:</label>
+    <input class="form-control" type="date" name="startDate" placeholder="example yyyy-mm-dd" required>
+
+    <label>End Date:</label>
+    <input class="form-control" type="date" name="endDate" placeholder="example yyyy-mm-dd" required>
+
+    <label>Start Time: </label>
+    <input class="form-control" type="text" name="startTime" placeholder="example 00:00" required>
+
+    <label>End Time: </label>
+    <input class="form-control" type="text" name="endTime" placeholder="example 00:00" required>
+
+    <label>Place: </label>
+    <li id="addPlaceIcon" class="glyphicon glyphicon-plus"></li>
+    <span class="addPlaceSpan text-muted">add new place</span>
+
+    <select id="selectAllPlaces" class="options form-control" name="placeId" required>
+      <option disabled selected>Select place..</option>
+      {{#each places}}
+        <option class="placesList" value="{{this.id}}">{{this.name}}</option>
+      {{/each}}
+    </select>
+
+    <div id="addPlaceContainer" class="add-place">
+      <input class="form-control" type="number" name="place_location" placeholder="location">
+      <input class="form-control" type="text" name="place_imageUrl" placeholder="image - link to image of place">
+      <select class="form-control" name="place_categories">
+        <option disabled selected>Select place categories..</option>
+        <option value="cafe">Cafe</option>
+        <option value="music venue">Music Venue</option>
+        <option value="religious site">Religious Site</option>
+        <option value="food and drink">Food and Drink</option>
+        <option value="education">Education</option>
+        <option value="retail">Retail</option>
+        <option value="sport">Sport</option>
+        <option value="municipal">Municipal</option>
+        <option value="healthcare">Healthcare</option>
+      </select>
+      <select multiple class="form-control" name="place_accessibilityOptions">
+        <option disabled selected>Select accessibility..</option>
+        <option value="Audio recordings">Audio Recordings</option>
+        <option value="Braille">Braille</option>
+        <option value="Big fonts">Big Fonts</option>
+        <option value="Carer">Carer</option>
+        <option value="Disabled parking">Disabled Parking</option>
+        <option value="Disabled toilets">Disabled Toilets</option>
+        <option value="Good lighting">Good Lighting</option>
+        <option value="Wheelchair access">Wheelchair Access</option>
```

Comment:
Still needs to be put in alphabetical order - wheelchair access, sms messaging and sign language are in the wrong order atm